basil.js

```diff
@@ -860,6 +860,48 @@ var clearConsole = function() {
 // Color
 // ----------------------------------------
 
+/**
+ * @summary Sets a background on the current page.
+ * @description Sets a background on the current page. The function takes the current canvas mode into account, so setting canvas mode to `FACING_PAGES` beforehand would result in the background to be drawn across the entire spread. The background will be placed on a background layer that basil creates automatically.
+ *
+ * @cat     Color
+ * @method  background
+ *
+ * @param   {Color|Gradient|Swatch|Numbers|String} bgColor Accepts a color/gradient/swatch as string name or variable. Or values: GRAY / R,G,B / C,M,Y,K.
+ * @return  {Rectangle} The newly created background shape.
+ */
+pub.background = function (bgColor) {
+
+  var pLayer = currentLayer();
+  var bgLayerName = "basil.js Background";
+  var bgLayer = pub.layer(bgLayerName);
+
+  var lockedState = bgLayer.locked;
+  bgLayer.locked = false;
+  pub.arrange(bgLayer, pub.BACK);
```

Comment:
I think it should be only arranged back if it newly created. When it already exists and there is a layer underneath it, it is probably done so on purpose.